### PR TITLE
Fix/issues

### LINKED ISSUE #470: [Update] Clarify that indexes are not used for compressed chunks
It would be really useful to update the docs to mention that indexes are not used when reading from compressed chunks - this is important info, but not mentioned anywhere.

**Location(s)**
https://docs.timescale.com/latest/using-timescaledb/compression

### PR DESCRIPTION
**Pull request by solugebefola**
PR instructions

1. Describe your PR right below:

Fixes various issues that were backlogged

Fixes #470 

---
2. Add label for the earliest DB version your changes apply to -->>

3. If you are changing page names or in-page anchors, you must update the
`page-index.js` file

4. Please add at least one content reviewer. If you don't add an
additional reviewer with knowledge about the area you are writing about,
one of the codeowners may add one

5. If this is in response to a posted GitHub issue, please add "Fixes <issue #>"
below so that it's referenced (i.e. "Fixes #122")

5. If you are making simple corrections, reviewers may add comments to your
PR without officially requesting changes. This is to avoid additional
request/review cycles for simple changes. Make sure to address these
comments (i.e. with changes) before your PR is ready to merge

6. Feel free to delete these instructions in your PR message after everything
else is filled out
---
Additional notes:

The default reviewers (CODEOWNERS) are added to each PR.  They will
primarily be reviewing on formatting, not content.  You only need ONE of them
to approve in order to merge your PR

### CODE REVIEW

File: using-timescaledb/hypertables.md [hunk 1]
```diff
@@ -126,7 +126,8 @@ taken if you make heavy use of expensive index types (e.g., some
 PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [`chunk_relation_size`][chunk_relation_size] function.
 
-**Space partitions**: Space partitioning is optional but can make
+**Space partitions**: [](best-practices-space-partitions)
```
Comment (mfreed) [inline using-timescaledb/hypertables.md:129]: Is this (best-practices-space-partitions) because it's an anchor link?

**Review by mfreed — APPROVED** [review]

**Review by svenklemm — APPROVED** [review]
LGTM

File: using-timescaledb/hypertables.md (hunk 1 above)
Comment (solugebefola) [reply using-timescaledb/hypertables.md:129]: yup

Comment (solugebefola) [reply using-timescaledb/hypertables.md:129]: More specifically, it's so I can directly reference this section from the `api.md`

**Pull request merged**